<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="DrawButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="0">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox1">
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="maxCount">
        <number>2147483645</number>
       </property>
       <item>
        <property name="text">
         <string>RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CMYK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HSL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XYZ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LAB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QSlider" name="horizontalSlider_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="29" column="1">
      <widget class="QLineEdit" name="lineEdit_2_4"/>
     </item>
     <item row="28" column="2">
      <widget class="QSlider" name="horizontalSlider_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QSlider" name="horizontalSlider_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox3">
       <item>
        <property name="text">
         <string>RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CMYK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HLS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XYZ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LAB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QSlider" name="horizontalSlider_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox2">
       <item>
        <property name="text">
         <string>RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CMYK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HLS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XYZ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LAB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="29" column="2">
      <widget class="QLineEdit" name="lineEdit_3_4"/>
     </item>
     <item row="28" column="1">
      <widget class="QSlider" name="horizontalSlider_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="28" column="0">
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="13" column="0">
      <widget class="QLineEdit" name="lineEdit_1"/>
     </item>
     <item row="20" column="2">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="19" column="0">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="29" column="0">
      <widget class="QLineEdit" name="lineEdit_1_4"/>
     </item>
     <item row="20" column="0">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="18" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="14" column="2">
      <widget class="QSlider" name="horizontalSlider_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="18" column="2">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="30" column="2">
      <widget class="QSlider" name="horizontalSlider_3_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="1">
      <widget class="QSlider" name="horizontalSlider_2_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="30" column="0">
      <widget class="QSlider" name="horizontalSlider_1_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
